stdlib/option.mli

```diff
@@ -54,6 +54,14 @@ val fold : none:'a -> some:('b -> 'a) -> 'b option -> 'a
 val iter : ('a -> unit) -> 'a option -> unit
 (** [iter f o] is [f v] if [o] is [Some v] and [()] otherwise. *)
 
+val for_all : ('a -> bool) -> 'a option -> bool
+(** [for_all p None] is [true],
+    [for_all p (Some v)] is [p v]. *)
+
+val exists : ('a -> bool) -> 'a option -> bool
+(** [exists p None] is [false],
+    [exists p (Some v)] is [p v]. *)
```

Comment:
The `@since` values for those two new functions are missing.

Reply:
Thanks! (I split this change in #13920 for better discussion.)